cpanfile

```diff
@@ -26,6 +26,34 @@ requires 'JSON::MaybeXS';
 recommends 'Cpanel::JSON::XS', '3.0206'; # 3.0206 adds 'allow_bignum' option
 recommends 'JSON::PP', '2.00'; # 1.99_01 adds 'allow_bignum'
 requires 'List::MoreUtils';
+requires 'Locale::CLDR';
+requires 'Locale::CLDR::Locales::Ar';
+requires 'Locale::CLDR::Locales::Bg';
+requires 'Locale::CLDR::Locales::Ca';
+requires 'Locale::CLDR::Locales::Cs';
+requires 'Locale::CLDR::Locales::Da';
+requires 'Locale::CLDR::Locales::De';
+requires 'Locale::CLDR::Locales::El';
+requires 'Locale::CLDR::Locales::En';
+requires 'Locale::CLDR::Locales::Es';
+requires 'Locale::CLDR::Locales::Et';
+requires 'Locale::CLDR::Locales::Fi';
+requires 'Locale::CLDR::Locales::Fr';
+requires 'Locale::CLDR::Locales::Hu';
+requires 'Locale::CLDR::Locales::Id';
+requires 'Locale::CLDR::Locales::Is';
+requires 'Locale::CLDR::Locales::It';
+requires 'Locale::CLDR::Locales::Lt';
+requires 'Locale::CLDR::Locales::Ms';
+requires 'Locale::CLDR::Locales::Nb';
+requires 'Locale::CLDR::Locales::Nl';
+requires 'Locale::CLDR::Locales::Pl';
+requires 'Locale::CLDR::Locales::Pt';
+requires 'Locale::CLDR::Locales::Ru';
+requires 'Locale::CLDR::Locales::Sv';
+requires 'Locale::CLDR::Locales::Tr';
+requires 'Locale::CLDR::Locales::Uk';
+requires 'Locale::CLDR::Locales::Zh';
```

Comment:
Is there an "all" placeholder? Explicitly naming them all is likely to be a maintenance issue.

Reply:
I certainly use Bundle::Locale::CLDR::Everything but that will pull at least 180 Locale::CLDR::Locales that we don't use

Reply:
Ok. In that case, I think we need to add cross references to each place we store them, mentioning that adding an element in one place requires adding it in .. and .. and ... other places.

Reply:
Documented in Pg-database.sql and cpanfile